controller-server/src/main/java/com/yahoo/vespa/hosted/controller/maintenance/ApplicationOwnershipConfirmer.java

```diff
@@ -81,6 +82,23 @@ private void ensureConfirmationResponses() {
             });
     }
 
+    private void updateConfirmedApplicationOwners() {
+        ApplicationList.from(controller().applications().asList())
+                .withProjectId()
+                .hasProductionDeployment()
+                .asList()
+                .stream()
+                .filter(application -> application.ownershipIssueId().isPresent())
+                .filter(application -> !application.owner().isPresent())
+                .forEach(application -> {
+                    IssueId ownershipIssueId = application.ownershipIssueId().get();
+                    ownershipIssues.getConfirmedOwner(ownershipIssueId).ifPresent(owner -> {
+                        controller().applications().lockIfPresent(application.id(), applicationId ->
```

Comment:
I think `lockedApplication` would be a better name for a locked application than `applicationId` ;)